jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -56,6 +58,7 @@ def __init__(self, *args, **kwargs):
         self._marks = {}
         self._dict = {}  # dictionary representation of current mouseover info
         self._x, self._y = None, None  # latest known cursor positions
+        self.current_unit = None
```

Comment:
this is the current units of data in images, right?  This is not used for any other viewer (spectrum, etc)?  Can it be named something more specific to avoid confusion?

Reply:
I like `image_unit` for now to avoid confusion with spectral cases.  Down the road we may need this to be a dictionary or a method that looks up per-viewer, but no need to introduce that complexity here.